src/lib/updateSiteUrlToLocal.ts

```diff
@@ -12,12 +12,15 @@ export const updateSiteUrlToLocal = async ( siteId: string ) => {
 	} );
 
 	if ( ! currentSiteUrl ) {
-		console.error( 'Failed to fetch site URL after import' );
+		console.error( 'Failed to fetch site URL' );
```

Comment:
Could we rename this file to follow kebab-case format as the other files in the project?